docs/guides/layout/Layout.stories.mdx

```diff
@@ -0,0 +1,110 @@
+        <th style={{ width: 150 }}>Gutters (px)</th>
+        <th style={{ width: 150 }}>Number of columns</th>
+      </tr>
+    </thead>
+    <tbody>
+      {Object.entries(getBreakpoints()).map((entry) => {
+        return (
+          <tr key={entry[0]}>
+            <td>{entry[0]}</td>
+            <td>{entry[1].value}</td>
+            <td>{entry[1].gutter}</td>
+            <td>{entry[1].column}</td>
+          </tr>
+        );
+      })}
+    </tbody>
+  </table>
+</>
+
+Whenever the screen resolution goes above or below a breakpoint, the layout should adapt itself to suit the screen size and for the content to be accessible.
+
+## Container <a id="container" />
+
+The container is one of the most basic layout components we provide to our community, and it enables you to center content horizontally on the page.
+
+This component is very useful when your content can't grow beyond a specific breakpoint. In this case, you can use `maxWidth` to specify the breakpoint
+to which the content is bounded. The content will automatically grow or shrink according with the size of the screen and breakpoint used ensuring that
+it doesn't grow beyond the specified width.
+
+If necessary, multiple nested containers can be used as shown in the example below.
+
+<Canvas>
+  <Story name="Container" story={ContainerSample} />
+</Canvas>
+
+For more information, please go to the [container's API](/docs/components-container--main).
+
+## Grid <a id="grid" />
+
+This component enables you to create responsive grid layouts with a variable amount of columns. The grid items will also adapt themselves
+to suit the screen sizes.
+
+By default the grid component is based on a 12-column grid layout. This can be easily changed through the `columns` property. If this property is set to
+`auto`, the Design System directives will be used where the number of columns will update depending on the breakpoint.
+
+The grid component also enables you to span items across columns if needed. Thus, with this component you'll be able to create a variety of layouts depending
+on your needs.
+
+Below is an example on how to use the grid component where the Design System directives were used and some grid items were spanned across multiple columns.
+
+<Canvas>
+  <Story name="Grid" story={GridSample} />
+</Canvas>
+
+Learn more about the grid component [here](/docs/components-grid-grid--main).
+
+## Simple Grid <a id="simple-grid" />
+
+The simple grid component is an alternative to the grid component and it provides an easy way of creating responsive grids where each item takes an equal amount
+of space. The items will grow or shrink whenever the screen size changes.
+
+This component should be used whenever you need to create grids where items should be distributed across a specific number of columns with the same width.
+
+This component also enables you to create your own breakpoints and define the spacing and number of columns for each one. Thus, contrary to the grid component,
```

Comment:
We should review this **later** - I don't think we should be promoting users to create their column own breakpoints

Reply:
Yes, what you are saying makes sense. I added this information because it seems like it's a feature provided by the simple grid and it distinguishes it from the grid. However, we should discuss this component in more detail and define what we actual want from it and promote to users. 
However, I'll change the example and use the DS specs and not some random breakpoints.